Solutions/Recorded Future/Playbooks/RecordedFuture-DOMAIN-C2_DNS_Name-IndicatorProcessor.json

```diff
@@ -25,8 +26,9 @@
 			"name": "[parameters('PlaybookName')]",
 			"location": "[resourceGroup().location]",
 			"dependsOn": [
-				"[resourceId('Microsoft.Web/connections', variables('RecordedFutureConnectionName'))]"
-			],
+				"[resourceId('Microsoft.Web/connections', variables('RecordedFutureConnectionName'))]",
+                "[resourceId('Microsoft.Logic/workflows', variables('RecordedFutureImportPlaybookName'))]"
```

Comment:
Deployment is failing as this resource is not present in the template. As we have already mentioned in the README file that this playbook is required as a pre-requisite, we should be good not mentioning this here.

Reply:
We have a dependency between this playbook and the RecordedFuture-ImportToSentinel.json where solution installations fail today in the published version that we need to sort out. 

I'm going to try this

``` json 
     "variables": {
        "RecordedFutureImportPlaybookName": "[parameters('PlaybookNameBatching')]"
```

``` json		
"dependsOn": [
	        "[resourceId('Microsoft.Web/connections', variables('RecordedFutureConnectionName'))]",
                "[resourceId('Microsoft.Logic/workflows', variables('RecordedFutureImportPlaybookName'))]"
```

Also how is the best way of testing a solution installation?

Reply:
@RecordedFutureOskbo Can probably assist with testing

Reply:
@RecordedFutureOskbo To test the deployment use the instructions mentioned here - https://github.com/Azure/Azure-Sentinel/tree/master/Playbooks#Instructions-for-deploying-a-custom-template.

I understand that we have requirement to install "RecordedFutureImportPlaybookName" playbook before but unless you have the template resource in the ARM template itself, dependsOn clause will not work. Currently, it searches for resource with the required name variables('RecordedFutureImportPlaybookName') within the resources array of this ARM template and since it is not present here, it fails. You can mention in the README clearly instructions on how to install the playbooks, the order and the pre-requisite at each step.